docs/administration/recent_activity/recent_activity.md

```diff
@@ -51,6 +51,14 @@ This policy is required to view [activity log in user profile]([[= user_doc =]]/
 
     Do not assign `activity_log/read` permission to the Anonymous role, even with the owner limitation, because this role is shared among all unauthenticated users.
 
+## User privacy
+
+When acting through HTTP, User's login name and IP are logged.
```

Comment:
```suggestion
When acting through HTTP, User's login name and IP address are logged.
```
If HTTPS, the login name may not be available but the IP will still be logged, won't it? It should perhaps be made more clear what is meant here.

Reply:
It's the Ibexa DXP current user login name. It could be `anonymous`. HTTPS shouldn't mask it.

```suggestion
The username of the User performing the action is logged. When acting through HTTP, User's IP is also logged.
```